src/lib/throttle.js

```diff
@@ -132,3 +136,40 @@ export function getReturnToken() : ?string {
         return token;
     }
 }
+
+export function buildFundingLogoThrottle(props : Object) : ?Throttle {
+
+    let { layout, label, locale, browserLocale } = props;
+
+    if (browserLocale.country !== COUNTRY.US) {
+        return null;
+    }
+
+    let localeString = locale;
+
+    if (typeof locale === 'object') {
+        localeString = `${ locale.lang }_${ locale.country }`;
+    }
+
+    if (localeString !== 'en_US') {
+        return null;
+    }
+
+    if (label !== BUTTON_LABEL.CHECKOUT && label !== BUTTON_LABEL.PAYPAL && label !== BUTTON_LABEL.PAY && label !== BUTTON_LABEL.BUYNOW) {
+        return null;
+    }
+
+    let domain = getDomain().replace(/^https?:\/\//, '').replace(/^www\./, '');
+    if (config.bmlCreditTest.domains.indexOf(domain) === -1) {
+        return null;
+    }
+
+    if (layout === undefined || (layout && layout === BUTTON_LAYOUT.HORIZONTAL)) {
+        return {
```

Comment:
Can probably just directly return `getThrottle('ppc_rebrand', 50)` here right?

Reply:
Oh yup!  That was a leftover from adding in the isActive flag to the object.